### DISCUSSION TITLE
Do I need a specific version of Kubernetes to install from version 0.22 -> 0.27

### DISCUSSION THREAD

**Discussion by Thomas-Rice** [strimzi/strimzi-kafka-operator#6185] [Q&A] [answered]
I am trying to upgrade my operator from version 0.22 to 0.27 but coming accross the following issue: 
![image](https://user-images.githubusercontent.com/9338637/149412003-c8972122-8c26-4b11-931b-2301cf8d9816.png)

I have done some searching of the web and checked that the CRD is at version 0.22 and supports v1beta2. So I am now wondering if my version of Kubernetes (1.19.3) might not have the new api changes to apiextensions.k8s.io/v1 and causing the issue.

**Comment by scholzj** [comment, marked as answer]
0.27.0 supports Kubernetes 1.16 and higher, so you should be fine. Have you done the API conversion using the provided tool as described in all the docs and release notes? If not, please have a look at it and give it a try.

**Reply by Thomas-Rice** [reply]
Thank you for the reply! 

The underlying issues for this was that the when running the conversion tool it would fail silently when running on VScode. 
I needed to install Java and have my JAVA_HOME var set in order to run the CRD upgrade feature. 
Once that was done the upgrade was possible.
